_data/simple-icons.json

```diff
@@ -1755,6 +1785,11 @@
             "hex": "F5C300",
             "source": "http://www.spreaker.com/press"
         },
+        {
+            "title": "Sprint",
+            "hex": "FFDD05",
+            "source": "https://commons.wikimedia.org/wiki/File:Logo_of_Sprint_Nextel.svg"
```

Comment:
There is an SVG version of their logo on their [landing page](https://www.sprint.com/) which uses the color `#FFCE0A` rather then `#FFDD05`, could you update both the color and the source (and possible the SVG as well, if it differs from the one found in the link)

Reply:
Can do!